README.md

```diff
@@ -128,8 +128,51 @@ Docker
 
 - [Developing minetestserver with Docker](doc/developing/docker.md)
 
-We provide a Dockerfile that can be used to build the server.
+We provide Minetest server Docker images using the GitHub container registry.
 
+Images are built on each commit and available using the following tag scheme:
+
+* `ghcr.io/minetest/minetest:master` (latest build)
+* `ghcr.io/minetest/minetest:<tag>` (specific tag)
+* `ghcr.io/minetest/minetest:latest` (latest tag)
+
+See [here](https://github.com/minetest/minetest/pkgs/container/minetest) for the available tags.
+
+If you want to test it on a Docker server you can easily run:
+
+```shell
+sudo docker run ghcr.io/minetest/minetest:master
+```
+
+If you want to use it in a production environment you should use volumes bound to the Docker host
+to persist data and modify the configuration:
+
+```shell
+sudo docker create -v /home/minetest/data/:/var/lib/minetest/ -v /home/minetest/conf/:/etc/minetest/ ghcr.io/minetest/minetest:master
+```
+
+You may also want to use [Docker Compose](https://docs.docker.com/compose):
+
+```yaml
+---
+version: "2"
+services:
+  minetest_server:
+    image: ghcr.io/minetest/minetest:master
+    restart: always
+    networks:
+      - default
+    volumes:
+      - /home/minetest/data/:/var/lib/minetest/
+      - /home/minetest/conf/:/etc/minetest/
+    ports:
+      - "30000:30000/udp"
+      - "30000:30000/tcp"
```

Comment:
not sure about the syntax but this should be bound to localhost so people don't accidentally expose their prometheus

Reply:
yes